src/pages/token-auth-rules/index.md

```diff
@@ -3,10 +3,42 @@ title: Overview
+## Introduction
+When a token operation is performed, the program can be called with instruction details (e.g. destination address for a token transfer), and those details are validated against the Rule Set. If the Rules are evaluated to be invalid, the instruction will fail and the whole transaction will be reverted. This enables whole transactions to be built that couple token operations with the Token Auth Rules program so any transactions, and therefore the contained token operation, will be reverted if the Rules in the associated Rule Set are violated.
 
-In the meantime, you may be interested in the following resources:
+## Features
+
+[Create or Update Rule Sets](/token-auth-rules/create-or-update) - An instruction used to both initialize and update Rule Set contents.
+
+[Rule Set Buffers](/token-auth-rules/buffers) - How large Rule Sets are handled.
+
+[Validate Rule Sets](/token-auth-rules/validate) - How a Rule Set is validated.
+
+## Rule Types
+Authorization rules are variants of a `Rule` enum that implements a `validate()` method.
+
+There are **Primitive Rules** and **Composed Rules** that are created by combining of one or more primitive rules.
+
+**Primitive Rules** store any accounts or data needed for evaluation, and at runtime will produce a true or false output based on accounts and a well-defined `Payload` that are passed into the `validate()` function.
+
+**Composite Rules** return a true or false based on whether any or all of the primitive rules return true.  Composed rules can then be combined into higher-level composed rules that implement more complex boolean logic.  Because of the recursive definition of the `Rule` enum, calling `validate()` on a top-level composed rule will start at the top and validate at every level, down to the component primitive rules.
+
+## Operation
+A Rule Set is built upon the `HashMap` data structure and is meant to store various sets of rules for different instruction types that could be used with a token (e.g. transfer, delegate, burn, etc.). Token Auth Rules uses the term **Operation** for these various instructions and **Operations** are used as keys in the `HashMap` data structure. Each **Operation** can have a different set of associated rules.
+
+### Scenario
+**Scenarios** are considered an extension of the **Operation** association and are used for the many different circumstances under which an instruction can be called. For example, Token Metadata uses the authority type as a **Scenario** for calls to Token Auth Rules from Token Metadata. A Transfer **Operation** may be triggered on a token by either the token's owner or delegate, and the Rule Set manager may want these different scenarios to be governed by different rules and so a **Scenario** can be used to manage this distinction. An **Operation** and **Scenario** combination is just two strings separated by a colon and so the two `HashMap` keys used for the prior example would be `Transfer:Owner` and `Transfer:Delegate`.
```

Comment:
This section is a bit hard to understand.  It gets easier once you read the sentence: "An **Operation** and **Scenario** combination is just two strings separated by a colon and so the two `HashMap` keys used for the prior example would be `Transfer:Owner` and `Transfer:Delegate`."  So like once you see it in action its clear what it is.

One idea would be moving the "An **Operation** and **Scenario** combination is just two strings..." sentence to be the second sentence of the paragraph?